illumos-utils/src/lib.rs

```diff
@@ -11,6 +11,7 @@ pub mod dladm;
 pub mod fstyp;
 pub mod link;
 pub mod opte;
+pub mod rsrvrctl;
```

Comment:
`rsrvrctl` is just a "cute" name for the Private utility.  I'd probably call this something like `vmm_reservoir`, since it's not bound by that

Reply:
Done